core/src/dird/vbackup.cc

```diff
@@ -564,12 +566,21 @@ static bool CreateBootstrapFile(JobControlRecord& jcr,
   rx.bsr = std::make_unique<RestoreBootstrapRecord>();
   rx.JobIds = jobids_pm.addr();
 
+  rx.found = false;
+
   if (DbLocker _{jcr.db_batch}; !jcr.db_batch->GetFileList(
           &jcr, rx.JobIds, false /* don't use md5 */, true /* use delta */,
-          InsertBootstrapHandler, (void*)rx.bsr.get())) {
+          InsertBootstrapHandler, (void*)&rx)) {
```

Comment:
Do we really want to pass the address of the pointer here?
Also, the cast to `void*` is implicit.